SPECS/systemd/systemd.spec

```diff
@@ -272,6 +272,7 @@ Provides:       /sbin/shutdown
 Provides:       syslog
 Provides:       systemd-units = %{version}-%{release}
 Obsoletes:      systemd-bootstrap <= %{version}-%{release}
+Provides:       systemd-bootstrap = %{version}-%{release}
```

Comment:
yeah it's unusual; the '*-bootstrap-*' packages are needed *only* due to the design of our toolkit. So the `Obsoletes:` is needed because our toolkit has the `*-bootstrap-*` packages installed but they install files that conflict with our actual, real packages (e.g. `systemd-bootstrap-rpm-macros` installs the exact same files as `systemd-rpm-macros`) so both can't be installed at the same time. So, once the toolkit gets around to trying to install conflicting packages (e.g. trying to install the `systemd-rpm-macros` package it just built), it has to remove the `bootstrap` version, which is why the real package includes `Obsoletes:` against the `bootstrap` version.

However, when building locally lots of our packages have (e.g.) `BuildRequires: systemd-bootstrap-rpm-macros`, and since that's not actually a real package (it's built, installed, and used *only* by our toolkit before it builds our real `systemd-rpm-macros` package), it's not possible to provide that requirement. Which is why the `Provides:` is needed.